pyccel/ast/builtins.py

```diff
@@ -408,55 +422,18 @@ class PythonList(PythonTuple):
     _is_homogeneous = True
     """ Represent lists in the code with dynamic memory management."""
     def __init__(self, *args, **kwargs):
```

Comment:
Do we still need to accept `**kwargs` here?

Reply:
No I don't think so. The `kwargs` object is not used in the old code either